packages/docs/content/foundation/typography/list.mdx

```diff
@@ -75,7 +75,7 @@ import loremIpsumGenerator from '../../../src/helpers/loremIpsumGenerator';
 ### Accessibility
 
 - When a list has a heading, consider adding an `id` attribute to the heading element, and an `aria-labelledby` attribute to the list element. Screen readers will then announce the heading text when users navigate to the list.
-- When using an unstyled list, `role="list"` should be added to the list element. <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#accessibility_concerns" target="_blank">Safari will not recognize it as a list</a> in the accessibility tree.</p>
```

Comment:
This was a typo that came from the `main` branch, which I had to fix in order to build statically.